lib/screens/employees/employee_details_view.dart

```diff
@@ -478,7 +478,7 @@ class _EmployeeDetailsViewState extends State<EmployeeDetailsView> {
                     alignment: Alignment.center,
                     child: Text(
                       getInitials(employee?.name ?? "",
-                          employee?.avatarAttachmentUrl == null ? 2 : 0),
+                          employee?.avatarAttachmentUrl == null ? 2 : 0).toUpperCase(),
```

Comment:
@sriLatha2328 toUpperCase function to be moved inside the getInitials function so that it handles the upper cases wherever used.